src/fiatconnect/ReviewScreen.tsx

```diff
@@ -36,15 +42,35 @@ import { Currency } from 'src/utils/currencies'
 
 type Props = NativeStackScreenProps<StackParamList, Screens.FiatConnectReview>
 
+const disclaimerIsNeeded = (
+  flow: CICOFlow,
+  quote: FiatConnectQuote,
+  localCurrency: LocalCurrencyCode
+) => {
+  const fiatType = quote.getFiatType()
+  const currencyEligibleForDisclaimer =
```

Comment:
I don't get this `currencyEligibleForDisclaimer` thing. Is there some reason we wouldn't want to show this disclaimer when the local currency differs from the fiat currency on the quote?

I suspect that this was an effort to comply with the last AC of the issue in Linear https://linear.app/valora/issue/ACT-530/disclaimer-for-users-with-different-app-currency-than-cash-out , but I wonder if that AC was taken the wrong way. Instead of geo-fencing the feature, maybe that AC was supposed to give an MVP set of locales this should work in, which we can work to expand later on if more eng effort is needed. 

Also, why only for cash out? If you try to send USD from an NGN bank account (for instance), and it deducts more NGN than you expect, it still seems warning-worthy.

Reply:
I realize that cash out is in the AC also but idk if it makes sense either

Reply:
Good points! After discussion on slack, I've made it so the PR verifies that the disclaimer can show up in the MVP locales but will also display whenever the local currency differs from the quote fiat currency.